app/assets/stylesheets/application.scss

```diff
@@ -27,6 +27,18 @@ header .gem-c-metadata {
   header {
     background: $govuk-blue;
 
+    // FIXME this is to fix link colours in the metadata component on a
+    // blue background. This can be removed once the component has been
+    // updated to accommodate such a situation
+    .gem-c-metadata a,
+    .gem-c-metadata a:link,
+    .gem-c-metadata a:hover,
+    .gem-c-metadata a:active,
+    .gem-c-metadata a:visited {
+      color: $white;
```

Comment:
should you use the govuk-frontend colours 

`govuk-colour('white')`

Reply:
Not sure if that'll work in this app, but I'll give it a go.

Reply:
It worked, thanks.